src/components/GradesTab/EditModal/OverrideTable/index.jsx

```diff
@@ -16,6 +16,11 @@ const GRADE_OVERRIDE_HISTORY_COLUMNS = [
   { label: 'Adjusted grade', key: 'adjustedGrade' },
 ];
 
+/**
+ * <OverrideTable />
+ * Table containing previous grade override entries, and an "edit" row
+ * with todays date, an AdjustedGradeInput and a ReasonInput
+ */
```

Comment:
Not part of the PR but it seems _really weird_ the way this table is set up and I want to whine about it: The `<AdjustedGradeInput/>`/`<ReasonInput/>` feel like this should just be replaced with a `<AdjustGradeControls/>` which makes up the final row of the table. Breaking out the extra controls seems too piecemeal.

Reply:
also: objects (or object-creator functions that aren't react components) can't be "connect-ed" to redux, which is kind of frustrating.